Add the 'Getting Started with TabPanel' tutorial (#2200)
DevExpress/devextreme-documentation: concepts/05 UI Components/TabPanel/01 Getting Started with TabPanel/20 Create Tabs.md

To create tabs, specify the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array. TabPanel generates a tab with a view for each item in this array. 

> To configure an individual tab, specify a tab's [title](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#title). Use the [icon](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#icon) property to specify an image for a tab. Refer to this property's description to see the available icons. You can also use the [badge](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#badge) property to create a badge with custom text as shown in code below. This code produces the following output:

Reviewer: > Refer to this property's description to see the available icons.

I expected the icons to be displayed in this topic but there's only a list that describes possible ways of specifying the icons. Anyway, I suggest that we refer to the code instead.

```suggestion
To configure an individual tab, specify a tab's [title](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#title). Use the [icon](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#icon) property to specify an image for a tab. The code below uses icons from the [DevExtreme icon library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library). You can also use the [badge](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#badge) property to create a badge with custom text:
```

> ![DevExtreme TabPanel: Customized Tabs](/images/UiWidgets/tab-panel-getting-started-customized-tabs.png)

Reviewer: It seems logical that the result produced by a code should be given after this code.

```suggestion
```

            </DxItem>
            <DxItem title="Notes" icon="comment">
                <!-- ... -->
            </DxItem>
            <DxItem title="Role" icon="isnotblank" badge="new">
                <!-- ... -->
            </DxItem>
        </DxTabPanel>
    </template>

    <script>
    import 'devextreme/dist/css/dx.light.css';

    import DxTabPanel, { DxItem } from "devextreme-vue/tab-panel";

    export default {
        components: {
            DxTabPanel,
            DxItem
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import 'devextreme/dist/css/dx.light.css';

    import TabPanel, { Item } from "devextreme-react/tab-panel";

    const App = () => {
        return (
            <TabPanel>
                <Item title="Employee" icon="floppy">
                    {/* ... */}
                </Item>
                <Item title="Notes" icon="comment">
                    {/* ... */}
                </Item>
                <Item title="Role" icon="isnotblank" badge="new">
                    {/* ... */}
                </Item>
            </TabPanel>
        );
    }

    export default App;

---

> 

Reviewer: ```suggestion

The code above produces the following output:

![DevExtreme TabPanel: Customized Tabs](/images/UiWidgets/tab-panel-getting-started-customized-tabs.png)

```

> You can also use the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) property to specify a custom template for all tabs. **items**.[tabTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) overrides the **itemTitleTemplate** property and allows you to specify a template for an individual tab. This approach is not shown in this tutorial but you can refer to this property's description for details.

Reviewer: ```suggestion
You can also use the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) property to specify a custom template for all tabs. **items**.[tabTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) overrides the **itemTitleTemplate** property and allows you to specify a template for an individual tab. This approach is not shown in this tutorial but you can refer to the property descriptions for details.
```

> To create tabs, specify the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array. TabPanel generates a tab with a view for each item in this array. 

Reviewer 2: ```suggestion
Populate the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array with data items. The TabPanel generates a tab with a view for each item in this array.
```

Author: Shouldn't we say about the purpose of this action first? How about the following:
To create tabs, populate the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array with data. The TabPanel generates...

> To configure an individual tab, specify a tab's [title](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#title). Use the [icon](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#icon) property to specify an image for a tab. The code below uses icons from the [DevExtreme icon library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library). You can also use the [badge](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#badge) property to create a badge with custom text:

Reviewer 2: ```suggestion
Specify a tab's [title](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#title) and/or [icon](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#icon) properties to configure an individual tab. The code below uses icons from the [DevExtreme icon library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library). You can also use the [badge](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#badge) property to create a badge with custom text.
```

                <!-- ... -->
            </DxItem>
        </DxTabPanel>
    </template>

    <script>
    import 'devextreme/dist/css/dx.light.css';

    import DxTabPanel, { DxItem } from "devextreme-vue/tab-panel";

    export default {
        components: {
            DxTabPanel,
            DxItem
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import 'devextreme/dist/css/dx.light.css';

    import TabPanel, { Item } from "devextreme-react/tab-panel";

    const App = () => {
        return (
            <TabPanel>
                <Item title="Employee" icon="floppy">
                    {/* ... */}
                </Item>
                <Item title="Notes" icon="comment">
                    {/* ... */}
                </Item>
                <Item title="Role" icon="isnotblank" badge="new">
                    {/* ... */}
                </Item>
            </TabPanel>
        );
    }

    export default App;

---

The code above produces the following output:

![DevExtreme TabPanel: Customized Tabs](/images/UiWidgets/tab-panel-getting-started-customized-tabs.png)

> You can also use the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) property to specify a custom template for all tabs. **items**.[tabTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) overrides the **itemTitleTemplate** property and allows you to specify a template for an individual tab. This approach is not shown in this tutorial but you can refer to the property descriptions for details.

Reviewer 3: ```suggestion
You can also use the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) property to specify a custom template for all tabs. **items**.[tabTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) overrides the **itemTitleTemplate** property and allows you to specify a template for an individual tab. This approach is not shown in this tutorial, but you can refer to the property descriptions for details.
```
